### PR TITLE
Bump SDKs to 3.8.0

### PR DESCRIPTION
**Pull request by ianlin-bbpos**
## Summary

Bump SDKs to 3.8.0.

## Motivation

Upgrade the native SDKs to the latest.

## Testing

- [x] I tested this manually
- [ ] I added automated tests

## Documentation

Select one:

- [ ] I have added relevant documentation for my changes.
- [x] This PR does not result in any developer-facing changes.

### CODE REVIEW

**Comment by bric-stripe** [comment]
we just today shipped an iOS 3.8.1 hotfix, could you update this to include that?

File: android/src/main/java/com/stripeterminalreactnative/Mappers.kt [hunk 1]
```diff
@@ -142,6 +142,8 @@ internal fun mapFromDeviceType(type: DeviceType): String {
         DeviceType.STRIPE_M2 -> "stripeM2"
         DeviceType.STRIPE_S700 -> "stripeS700"
         DeviceType.STRIPE_S700_DEVKIT -> "stripeS700Devkit"
+        DeviceType.STRIPE_S710 -> "stripeS710"
```
Comment (nazli-stripe) [inline android/src/main/java/com/stripeterminalreactnative/Mappers.kt:145]: are these not needed for iOS?

Comment (ianlin-bbpos) [reply android/src/main/java/com/stripeterminalreactnative/Mappers.kt:145]: yes but iOS does't support S710 yet by v3.8.0.

**Review by nazli-stripe — APPROVED** [review]

**Comment by nazli-stripe** [comment]
@ianlin-bbpos this can be a different PR but 3.8.0 has support for Tap to Pay UX configuration. Can we add that as well? https://github.com/stripe/stripe-terminal-android/blob/master/CHANGELOG.md#tap-to-pay-localmobile

**Comment by ianlin-bbpos** [comment]
> @ianlin-bbpos this can be a different PR but 3.8.0 has support for Tap to Pay UX configuration. Can we add that as well? https://github.com/stripe/stripe-terminal-android/blob/master/CHANGELOG.md#tap-to-pay-localmobile

Copy that.

**Pull request merged**